modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/dto/v1_0/CartItemDTOConverter.java

```diff
@@ -95,6 +98,24 @@ public CartItem toDTO(DTOConverterContext dtoConverterContext)
 		};
 	}
 
+	private String _getErrorMessage(
+		CommerceOrderItem commerceOrderItem, Locale locale) {
+
+		String error = null;
```

Comment:
@luca-pellizzon we can call the CommerceOrderValidator so we can check all

Reply:
Here we are considering the single OrderItem, the CommerceOrderValidator is used later in the flow.